Assets/Prefabs/UI/Menus/LobbySelect.prefab

```diff
@@ -774,6 +1422,53 @@ RectTransform:
   m_CorrespondingSourceObject: {fileID: 5877506774103653876, guid: e472bbf090cca566cae93eec176f8dff, type: 3}
   m_PrefabInstance: {fileID: 1374497080081520336}
   m_PrefabAsset: {fileID: 0}
+--- !u!1 &8157740322213297794 stripped
+GameObject:
+  m_CorrespondingSourceObject: {fileID: 7072107538278606930, guid: e472bbf090cca566cae93eec176f8dff, type: 3}
+  m_PrefabInstance: {fileID: 1374497080081520336}
+  m_PrefabAsset: {fileID: 0}
+--- !u!114 &4914463133787348733
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 8157740322213297794}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d0b148fe25e99eb48b9724523833bab1, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  m_Delegates:
+  - eventID: 9
+    callback:
+      m_PersistentCalls:
+        m_Calls:
+        - m_Target: {fileID: 3583328391671930842}
+          m_TargetAssemblyTypeName: LobbySelect, Assembly-CSharp
+          m_MethodName: SetTitle
+          m_Mode: 5
+          m_Arguments:
+            m_ObjectArgument: {fileID: 0}
+            m_ObjectArgumentAssemblyTypeName: UnityEngine.Object, UnityEngine
+            m_IntArgument: 0
+            m_FloatArgument: 0
+            m_StringArgument: Queue
+            m_BoolArgument: 0
+          m_CallState: 2
+        - m_Target: {fileID: 3583328391671930842}
+          m_TargetAssemblyTypeName: LobbySelect, Assembly-CSharp
+          m_MethodName: SetDescription
+          m_Mode: 5
+          m_Arguments:
+            m_ObjectArgument: {fileID: 0}
+            m_ObjectArgumentAssemblyTypeName: UnityEngine.Object, UnityEngine
+            m_IntArgument: 0
+            m_FloatArgument: 0
+            m_StringArgument: Automatically search for other players! If you become
+              the host, remember to start the game!
```

Comment:
Perhaps something more before "if you become the host", like "Joins an open lobby if present, otherwise hosts a new one."

Reply:
I don't think its necessary to explain the underlying logic to players, I think some people might find that wording more confusing even

Reply:
Mkay, we'll see how this fares then 👍